string_manipulation.rb

```diff
@@ -243,6 +256,9 @@ def palindrome_check(my_phrase)
   return true
 end
 
+# Time complexity is O(n), the loop will run n times depending on the size of my_phrase
```

Comment:
Correct! The loop will run precisely n/2 times and hence, in big O terms, it will have a time complexity of O(n)